ios/RNCameraRoll.m

```diff
@@ -11,14 +11,36 @@
 #import “React/RCTBridge.h” // Required when used as a Pod in a Swift project
 #endif
 
+@interface AssetData : NSObject
```

Comment:
This class is just to hold information about the asset, orientation, and path where we write it to jpeg on disk.